components/PostArticle.vue

```diff
@@ -94,7 +100,14 @@ export default {
       })
     }
   }
+  // created() {
```

Comment:
Same as above, keep if there is a reason to keep it

Reply:
removed, does nothing, was for debug.